node/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "@wolox/eslint-config-node",
```

Comment:
@damfinkel right now, node projects refer to this package using `eslint-config-wolox` or `eslint-config-wolox-node`; does this mean we will need to change the dependency reference in all projects?

Reply:
nope, we'll keep the old package so the projects won't break, but it will be desirable that projects eventually migrate to this config, since all changes and updates will be made here. `eslint-config-node`, as well as all the other old configs won't be maintained anymore